packages/brick_offline_first/lib/src/offline_queue/offline_graphql_request_queue.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:async';
+
+import 'package:brick_offline_first/src/offline_queue/offline_queue_graphql_client.dart';
+import 'package:brick_offline_first/src/offline_queue/offline_request_queue.dart';
+import 'package:gql_exec/gql_exec.dart';
+import 'package:logging/logging.dart';
+
+class OfflineGraphqlRequestQueue extends OfflineRequestQueue<OfflineQueueGraphqlClient> {
```

Comment:
For consistency with `GraphqlRequest`. The file should be renamed too
```suggestion
class GraphqlOfflineRequestQueue extends OfflineRequestQueue<OfflineQueueGraphqlClient> {
```